camelComponent/build.gradle

```diff
@@ -21,6 +21,10 @@ ext {
     commonsLang3Version = '3.12.0'
 }
 
```

Comment:
build configuration that includes unit test components.  This allows connector tests to use them.